packages/compass-explain-plan/src/components/explain-tree/clock.tsx

```diff
@@ -105,10 +103,12 @@ function drawElapsedTimes({
     },
   ];
 
+  const ARC_STROKE_WIDTH = 4;
+
   arcs.forEach(({ startAngle, endAngle, fill }) => {
     const svgArc = d3.svg
       .arc()
-      .innerRadius(radius - strokeWidth)
+      .innerRadius(radius - ARC_STROKE_WIDTH)
       .outerRadius(radius)
```

Comment:
In the new design elapsed time seem to sit in the middle of the circle line, not fully inside of it:

![image](https://github.com/mongodb-js/compass/assets/5036933/d8b322d6-c726-428e-a43a-d67574ce4bac)